python/beeai_framework/backend/chat.py

```diff
@@ -113,6 +114,10 @@ def merge(self, other: Self) -> None:
         elif other.usage:
             self.usage = other.usage.model_copy()
 
+    def get_tool_calls(self) -> list[MessageToolCallContent]:
```

Comment:
FYI I also added a simpler version this in #498 so whichever merges second will need to address the conflict by removing mine.